src/widgets/Header/index.tsx

```diff
@@ -1,22 +1,74 @@
 'use client';
-import s from './style.module.scss';
-import Logo from '@/assets/images/logo.svg';
+import { clsx } from 'clsx';
+import Link from 'next/link';
+import { useState, useCallback } from 'react';
+
+import { CancelIcon } from '@/assets/icons/CancelIcon';
 import LogoBlue from '@/assets/images/logo-blue.svg';
-import LogoSmall from '@/assets/images/logo-small.svg';
-import { Container } from '@/components/Container';
+import LogoMobile from '@/assets/images/logo-mobile.svg';
+import Logo from '@/assets/images/logo.svg';
+import Hamburger from '@/assets/images/hamburger.svg';
+import HamburgerBlue from '@/assets/images/hamburger-blue.svg';
+
 import { Button } from '@/components/Button';
-import { HamburgerIcon } from '@/assets/icons/HamburgerIcon';
-import { clsx } from 'clsx';
+import { Container } from '@/components/Container';
 import { NavLink } from '@/components/NavLink';
-import Link from 'next/link';
+
+import s from './style.module.scss';
+import SocialMedia from '@/widgets/SocialMedia';
 
 interface HeaderProps {
   whiteBg?: boolean;
   isHome?: boolean;
 }
 
+type NavItem = {
+  href: string;
+  label: string;
+};
+
+const navItems: NavItem[] = [
+  {
+    href: 'about',
+    label: 'About Us',
+  },
+  {
+    href: 'services',
+    label: 'Services',
+  },
+  {
+    href: 'showcase',
+    label: 'Showcase',
+  },
+  // {
+  //   href: 'careers',
+  //   label: 'Careers',
+  // },
+  // {
+  //   href: 'open-source',
+  //   label: 'Open Source',
+  // },
+  // {
+  //   href: 'blogs',
+  //   label: 'Blogs',
+  // },
+];
+
+const navItemsMobile: NavItem[] = [
+  {
+    href: '',
+    label: 'Home page',
+  },
+  ...navItems,
+];
+
 export const Header = ({ whiteBg = false, isHome = false }: HeaderProps) => {
-  // TODO: Hamburger iconu whiteBg
+  const [openMobileMenu, setOpenMobileMenu] = useState(false);
+  const toggleMobileMenu = useCallback(() => {
```

Comment:
- Ideally, we should not scroll to the top of the current page when opening the mobile nav. It should just open over the top of the current content.
- We should not be able to scroll the page while the mobile nav is open